pkg/storaged/stratis-details.jsx

```diff
@@ -438,10 +437,8 @@ export const StratisPoolDetails = ({ client, pool }) => {
                     </>
                 ),
             }}>
-                <CardTitle>
-                    <Text component={TextVariants.h2}>
-                        {fmt_to_fragments((pool.Encrypted ? _("Encrypted Stratis pool $0") : _("Stratis pool $0")), <b>{pool.Name}</b>)}
-                    </Text>
+                <CardTitle component="h2">
+                    {fmt_to_fragments((pool.Encrypted ? _("Encrypted Stratis pool $0") : _("Stratis pool $0")), <b>{pool.Name}</b>)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18746-20230502-181707-d585bac8-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)